src/Makefile.am

```diff
@@ -23,6 +31,7 @@ LIBBITCOIN_CONSENSUS=libbitcoin_consensus.a
 LIBBITCOIN_CLI=libbitcoin_cli.a
 LIBBITCOIN_UTIL=libbitcoin_util.a
 LIBBITCOIN_CRYPTO_BASE=crypto/libbitcoin_crypto_base.a
+LIBBITCOINCRYPTO_BASE=crypto/libbitcoin_crypto_base.la
```

Comment:
Since e51ead8496f010315b21fb48465b120e5cb8e334 we're no longer going to be building both `.a`'s and `.la`'s, only `.la`'s, which circumvents this problem.

I do agree that naming can be clearer here so am going to followup with a PR that renames all our `.la` variables to `LTLIB*`.